hawkular-inventory-parent/hawkular-inventory-service/src/main/java/org/hawkular/inventory/service/InventoryServiceIspn.java

```diff
@@ -210,8 +213,20 @@ public void deleteResourceType(String type) {
         try {
             byte[] encoded = Files.readAllBytes(configPath.resolve(fileName));
             return Optional.of(new String(encoded, "UTF-8"));
-        } catch (IOException e) {
-            return Optional.empty();
+        } catch (IOException ioe) {
+            try {
+                InputStream is = this.getClass().getClassLoader().getResourceAsStream(fileName);
+                String text = null;
+                try (Scanner scanner = new Scanner(is, StandardCharsets.UTF_8.name())) {
```

Comment:
I guess the same way of reading should be used while reading at the initial location